src/main/java/core/basesyntax/service/impl/WriteServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriteService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteServiceImpl implements WriteService {
+
+    @Override
+    public void writeToFile(String toFile, String report) {
+        File file = new File(toFile);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create the file " + toFile, e);
+        }
```

Comment:
you don't need this block of code, cause `new FileWriter(file)` does the same (read Javadoc)
<img width="633" alt="image" src="https://github.com/mate-academy/jv-fruit-shop/assets/32339836/58d0c1a3-1d98-4412-b7b1-91b7d50fb33e">